ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -241,7 +248,7 @@ Get Model Inference
     ...    ${project_title}=${NONE}
     ${self_managed} =    Is RHODS Self-Managed
     ${url}=    Get Model Route via UI    ${model_name}
-    ${curl_cmd}=     Set Variable    curl -s ${url} -d ${inference_input}
+    ${curl_cmd}=     Set Variable    curl -sk ${url} -d @${inference_input}
```

Comment:
you can use the self-signed certificate instead, check the model serving (MM) suite

Reply:
Could  it be done later as an enhancement, we need to add this to smoke so that we can find issues in the RC builds.

Reply:
@bdattoma Removed the k tag now